### ISSUE TITLE
Agree a manual accessibility testing routine for all SciTools repos

### ISSUE THREAD

**Issue by trexfeathers** [SciTools/.github#213]
Automated testing and continuous integration - #214 - can do much heavy lifting in the accessibility space, but not all elements of accessibility testing should be automated. E.g. the correct alt-text will depend on context. The SciTools team should agree on appropriate manual testing e.g. keyboard navigation of headings or screen-reading a page, with help from experts at UK Met Office, and agree on a schedule for performing this.
